scaladoc/src/dotty/tools/scaladoc/tasty/comments/MemberLookup.scala

```diff
@@ -108,7 +108,7 @@ trait MemberLookup {
     given dotc.core.Contexts.Context = quotes.asInstanceOf[scala.quoted.runtime.impl.QuotesImpl].ctx
     val sym = rsym.asInstanceOf[dotc.core.Symbols.Symbol]
     // note: Predef has .info = NoType for some reason
-    sym.isCompleted && sym.info.exists
+    (sym.isCompleted && sym.info.exists) || sym.infoOrCompleter.isInstanceOf[dotty.tools.dotc.core.ClassfileLoader]
```

Comment:
Nice! Can you test if the reverse logic works? I.e. instead of checking if `sym.isCompleted`, check if `.infoOrCompleter` is instance of `DottyUnpickler`. (I think that's the correct class.) It's better to blacklist loaders we know to misbehave, I think.

Reply:
Good idea with blacklisting, will check that one tomorrow